src/Components/App/App.js

```diff
@@ -86,6 +85,7 @@ class App extends Component {
                   showFeatured={this.showFeatured}
                   handleSearchEntry={this.handleSearchEntry}
                   searchByWord={this.searchByWord}
+                  disabled={!this.state.value}
```

Comment:
We won't need this disabled prop

Reply:
Lets remove it on the next PR - my branch is all weird because of the merge conflicts